source/utils/Helper.cpp

```diff
@@ -274,11 +274,16 @@ namespace Helper
     {
         return TTostring(d);
     }
-    #ifndef __x86_64
+    #if !defined(__LP32__) || defined(__OpenBSD__)
     std::string toString(size_t d)
     {
         return TTostring(d);
     }
+    #else
+    std::string toString(unsigned long long d)
```

Comment:
Unused on non-OpenBSD and may break on build on 32bit platforms where `uint64_t` is `unsigned long long` but `toString(uint32_t)` is already defined earlier.